modules/customize-certificates-add-service-serving-mutating-webhook.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-mutating-webhook_{context}"]
+= Add the service CA bundle to a MutatingWebhookConfiguration
+
+You can annotate a MutatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+
+.Procedure
+
+. Annotate the MutatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true`:
++
+----
+$ oc annotate mutatingwebhoookconfigurations <mutatingwebhook-name> \//<1>
+     service.beta.openshift.io/inject-cabundle=true
+----
+<1> Replace `<mutatingwebhook-name>` with the name of the MutatingWebhookConfiguration to annotate.
++
+For example, use the following command to annotate the MutatingWebhookConfiguration `foo`:
++
+----
+$ oc annotate mutatingwebhoookconfigurations foo service.beta.openshift.io/inject-cabundle=true
+----
+
+. View the MutatingWebhookConfiguration to ensure that the certificate has been generated:
++
+----
+$ oc get mutatingwebhookconfigurations <mutatingwebhook-name> -o yaml
+----
++
+It is displayed as `caBundle` in the YAML output:
```

Comment:
The CA bundle will appear in the `clientConfig.caBundle` field of all webhooks in the YAML output.